ADR/0039-send-cloud-events.md

```diff
@@ -0,0 +1,34 @@
+# 40. Konflux should send cloud events all system events.
+
+Date: 2024-09-24
+
+## Status
+
+Proposed
+
+## Context
+
+Konflux had made the architectural decision to not use cloud events. However, that does not mean that
+Konflux should not emit cloud events.
+
+Emitting cloud events would allow Konflux users to easily track what is happening in the system. In addition,
+they can use these cloud events to create their own product-specific infrastructure to support their build
+and release process.
+
+To support this, all Konflux components should be required to emit cloud events for signicant events. These
```

Comment:
Let's consider if "should be required to emit" is too strong of a statement. I believe "should emit" better describes the intent.

Reply:
I agree on the "softening"